tokio-trace/src/span.rs

```diff
@@ -690,10 +691,10 @@ impl<'a> Drop for Entered<'a> {
 // ===== impl EnteredUnscoped =====
 
 impl EnteredUnscoped {
-    /// Consumes the entered guard and exits the span, returning a `Span`
-    /// handle that may then be entered again.
+    /// Consumes the entered guard and exits the span, returning a handle to the
+    /// previously entered `Span` that may then be entered again.
     pub fn exit(mut self) -> Span {
-        let span = self.span.take().expect("must be moved to exit");
+        let span = self.span.take().expect("span should only be taken once");
```

Comment:
I think you can be _even_ more helpful here by saying "span should only be taken on drop"

Reply:
(it's also taken on exit)

Reply:
Yeah, technically the message is something like: "span is taken only here and on drop. since we own self, neither can be the case", but that seems too long.

Reply:
Quite. I was trying to find a concise way to say that.